packages/studio-plugin/src/orchestrators/ParsingOrchestrator.ts

```diff
@@ -214,7 +214,14 @@ export default class ParsingOrchestrator {
         `The pages directory does not exist, expected directory to be at "${this.paths.pages}".`
       );
     }
-    return createFilenameMapping(this.paths.pages, this.getOrCreatePageFile);
+
+    const excludeReservedPagesJSFiles = (filename: string) =>
```

Comment:
Do we only want this filter if `isPagesJSRepo` is true?

Reply:
Updated!